Bottom: 8537237f483359a89583a620516e70c9f7a29b29
Top:    0df330848f9a1f4a75f1591264e007f5b7e3cc14
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-26 07:20:28 +0100

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index 32036d7..4f24674 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -188,7 +188,7 @@ DrawStrippedText (ElementTypePtr Element, int min_width)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   if (doing_pinout || doing_assy)
